### ISSUE TITLE
Question: Does it work with .net MAUI?

### ISSUE THREAD

**Issue by lnaie** [reactiveui/refit#1370]
Hello,
It's not stated anywhere while looking briefly around.
So... does it work with .net MAUI? :)

**Comment by hahyes** [comment]
Why not? It is the same .NET, just different GUI framework. It uses HttpClient anyway, so it should work almost everywhere just like .NET.

**Comment by rdavisau** [comment]
I've used it on MAUI, it works.

**Comment by lnaie** [comment]
Ok, guys. Thanks.

**Comment by glennawatson** [comment]
Yes, Refit works on .NET MAUI (confirmed by users in this thread). It's plain HttpClient under the hood. For MAUI/iOS and any AOT/trimmed target, use the source-generated client (the default) plus source-generated System.Text.Json metadata for best trimming/AOT behavior.

Closing as a usage question - for follow-ups please use Discussions: https://github.com/reactiveui/refit/discussions


<blockquote><img src="https://opengraph.githubassets.com/7da521d756b94831ff31d8d5fc75ecbe2c2d69a69928da79595847192e775278/reactiveui/refit" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/reactiveui/refit/discussions">reactiveui refit · Discussions</a></strong></div><div>Explore the GitHub Discussions forum for reactiveui refit. Discuss code, ask questions & collaborate with the developer community.</div></blockquote>
